.github/workflows/release-app.yml

```diff
@@ -154,12 +154,8 @@ jobs:
 
       - name: Deploy production api
         if: inputs.app == 'api'
-        uses: ./.github/workflows/deploy-production-api.yml
-        with:
-          tag: ${{ steps.tag.outputs.image-tag }}
+        run: "gh workflow run 'Deployment: production-api' -f tag=${{ steps.tag.outputs.image-tag }}"
 
       - name: Deploy production frontend
         if: inputs.app == 'frontend'
-        uses: ./.github/workflows/deploy-production-nuxt.yml
-        with:
-          tag: ${{ steps.tag.outputs.image-tag }}
+        run: "gh workflow run 'Deployment: production-nuxt' -f tag=${{ steps.tag.outputs.image-tag }}"
```

Comment:
I think part of the reason we couldn't simplify this previously was that we couldn't dynamically determine which action to use - now that this is just running an arbitrary `gh workflow run` command, perhaps there's some room to consolidate here!

Reply:
Oh right, we could. We'd still have to translate `frontend` to `nuxt`, but if we wanted to collapse it into a single line it's possible now for sure.

Reply:
Keeping them separate has its advantages.

- It's clearer because they have different dependent jobs and required checks.
- It's easier to report their outcome independently in #1066.
- The code with matrices might end up being almost the same number of lines.